src/com/amplitude/Event.java

```diff
@@ -3,7 +3,104 @@
 import org.json.simple.JSONObject;
 
 public class Event {
+    // should move into Constants.java
+    public static final int MAX_PROPERTY_KEYS = 1024;
+    public static final int MAX_STRING_LENGTH = 1000;
+    public static final String TAG = "com.amplitude.Event"; //AmplitudeClient.class.getName();
 
-    private JsonObject eventProps;
+    private JSObject event;
+
+    public Event(String eventName, JSONObject eventProps, JSONObject groups, JSONObject groupProps,
```

Comment:
What is JSONObject groups and JSONObject groupProps? I feel like it's not needed for beta right now